python/tvm/driver/tvmc/__init__.py

```diff
@@ -14,3 +14,8 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+"""
+TVMC - TVM driver command-line interface
+"""
+
+from .compiler import add_compile_parser
```

Comment:
It looks weird to expose `add_compile_parser`. If your just want to trigger the decorator for registration, `import compile` should be sufficient?

Reply:
Makes sense. Just to be certain I'm getting the compiler module I want, I'll change it to `from . import compiler`.